src/main/java/org/jabref/logic/importer/FetcherResult.java

```diff
@@ -0,0 +1,15 @@
+package org.jabref.logic.importer;
+
+import java.net.URL;
+
+import org.jabref.logic.importer.fetcher.TrustLevel;
+
+public final class FetcherResult {
+    public final TrustLevel trust;
```

Comment:
Hide fields and make them accessible via getter?

Reply:
Ok, will do so.